ec_tools/semi_integration.py

```diff
@@ -25,19 +26,20 @@ def prepare_kernel(q, Δx, N, c1, c2):
     # auxiliary array
     s = np.zeros(n_filters)
 
-    for i in range(2*c1*c2):
+    for i in range(2 * c1 * c2):
         j = i - c1 * c2
         a_j = (a0 / c2) * np.exp(j / c2)
-        t_j = τ0 * np.exp(-j / (q*c2))
-        w1[i] = t_j / (Δx + t_j)
+        t_j = tau0 * np.exp(-j / (q * c2))
+        w1[i] = t_j / (delta_x + t_j)
         w2[i] = a_j * (1 - w1[i])
     return s, w1, w2
 
-def semi_integration(y, q=-0.5, Δx=1, c1=8, c2=2):
+
+def semi_integration(y, q=-0.5, delta_x=1, c1=8, c2=2):
     """
     Return the semiintegral R of order q for y with the x interval Δx and the filter constants
     c1 and c2.
-    
+
     Semi-integrating two times with order q = -0.5 should give the same result as integrating once.
     The relative error should not exceed 0.25 percent for 1000 and 0.5 percent per 10000 integration steps.
     TODO:
```

Comment:
Maybe you should open an issue for the Todo and add the issue number
```suggestion
    TODO:: see #X
```

Reply:
```suggestion
    TODO:: see #6
```